source/features/sync-pr-commit-title.tsx

```diff
@@ -84,11 +83,11 @@ function disableSubmission(): void {
 	getUI().remove();
 }
 
-const listeners: Deinit[] = [];
+const subscriptions: delegate.Subscription[] = [];
 
-function init(): Deinit {
-	listeners.push(
-		...onPrMergePanelOpen(updateCommitTitle),
+function init(signal: AbortSignal): Deinit {
+	subscriptions.push(
+		onPrMergePanelOpen(updateCommitTitle, signal),
 		delegate(document, '#merge_title_field', 'input', updateUI),
 		delegate(document, 'form.js-merge-pull-request', 'submit', updatePRTitle),
 		delegate(document, '.rgh-sync-pr-commit-title', 'click', disableSubmission),
```

Comment:
Worth it to pass the signal to `disableSubmission` so we can replace the custom deinit with a simple `signal.abort()`?

Reply:
Up to you. The alternative could also be to use `subscriptions.push(signal)` with the global `signal` so we'd still have the custom deinit but it would be simpler